packages/frontend/src/helpers/isFieldHidden.ts

```diff
@@ -0,0 +1,23 @@
+import type { IField, IJSONObject } from '@plumber/types'
+
+import get from 'lodash.get'
+
+export function isFieldHidden(
+  hiddenIfCondition: IField['hiddenIf'] | null | undefined,
```

Comment:
can I ask if this could be an optional argument instead so that it is `hiddenIfCondition?: IField['hiddenIf']` and then the function `isFieldHidden` will just take in this `IFieldVisibilityCondition` as an optional argument instead?

Reply:
good question - I explicitly didn't want the argument to be optional because it doesn't make sense semantically (there's no point calling `isFieldHidden` if you don't have a visibility condition to check against)

The reason I added `undefined` is really just-in-case. TBH `undefined` should not be possible, but with our lack of strict mode, I wasn't too sure.